main.go

```diff
@@ -51,11 +57,11 @@ func Archive(path string) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	arc := &txtar.Archive{}
+	var arc []file
 	for _, f := range files {
```

Comment:
I don't know if we care about this,  but it might be better to use `for i:=0; i<len(files); i++ { ...` here because it doesn't copy the entire file data.